atlasdb-config/src/main/java/com/palantir/atlasdb/http/FailoverFeignTarget.java

```diff
@@ -143,11 +147,16 @@ private void checkAndHandleFailure(RetryableException ex) {
     }
 
 
-    private void pauseForBackOff(RetryableException ex) {
-        double pow = Math.pow(
+    private void pauseForBackoff(RetryableException ex) {
+        double pauseTime = Math.pow(
                 GOLDEN_RATIO,
                 numSwitches.get() * failuresBeforeSwitching + failuresSinceLastSwitch.get());
-        long timeout = Math.min(maxBackoffMillis, Math.round(pow));
+        pauseForBackoff(ex, Math.round(pauseTime));
```

Comment:
may want to incorporate some jitter in the backoff, see https://www.awsarchitectureblog.com/2015/03/backoff.html